tests/karate/src/test/java/fe/utils/web.feature

```diff
@@ -89,7 +99,7 @@ Feature: web test
     * script("setTimeout(() => document.evaluate('//div[text()=\\'Create Roll-Call\\']', document, null, XPathResult.FIRST_ORDERED_NODE_TYPE, null).singleNodeValue.click(), 1000)")
 
     # Provide roll call required information
-    And retry(5,1000).input(roll_call_title_selector, constants.RC_NAME)
+    And retry(5, 1000).input(roll_call_title_selector, constants.RC_NAME)
```

Comment:
If you want to write the retry() that way, you should change it everywhere.

(I think it is better like that 👍)

Reply:
If I do it, it will be in a subsequent PR